apps/email/js/sync.js

```diff
@@ -152,8 +150,16 @@ define(function(require) {
         model.latestOnce('account', function(currentAccount) {
           fetchNotificationsData('sync').then(
             function(existingNotificationsData) {
-              mozL10n.formatValue('senders-separation-sign').then(separator => {
-                fn(app, currentAccount, existingNotificationsData, separator);
+              mozL10n.formatValue('senders-separation-sign')
+              .then(function(separator) {
+                var localized = {
+                  separator
+                };
+                mozL10n.formatValue('notification-no-subject')
```

Comment:
Ahh yes, construction resumes on the pyramid of doom... moohoohahaha!

(Referencing the comment above.  This is fine.  Maybe we can make things happier in convoy.)